deals.go

```diff
@@ -15,15 +15,31 @@ const (
 
 	DealTypeIncome  = "income"
 	DealTypeExpense = "expense"
+	DealStatusSettled = "settled"
+	DealStatusUnsettled = "unsettled"
 )
 
+type DealsResponse struct {
+	Deals []Deal `json:"deals"`
+}
+
 type DealResponse struct {
 	Deal Deal `json:"deal"`
 }
 
 type GetDealOpts struct {
+	// 取引先ID
+	PartnerID int32 `json:"partner_id,omitempty" url:"partner_id,omitempty"`
+	// 決済状況 (未決済: unsettled, 完了: settled)
+	Status string `json:"status,omitempty" url:"status,omitempty"`
+	// 収支区分 (収入: income, 支出: expense)
+	Type string `json:"type,omitempty" url:"type,omitempty"`
+	// 発生日で絞込：開始日(yyyy-mm-dd)
+	StartIssueDate string `json:"start_issue_date,omitempty" url:"start_issue_date,omitempty"`
+	// 発生日で絞込：終了日(yyyy-mm-dd)
+	EndIssueDate string `json:"end_issue_date,omitempty" url:"end_issue_date,omitempty"`
 	// 取引の債権債務行の表示（without: 表示しない(デフォルト), with: 表示する）
-	Accruals string `url:"accruals,omitempty"`
+	Accruals string `url:"accruals,omitempty" url:"accruals,omitempty"`
```

Comment:
duplicate struct tag:`url`

Reply:
`Get*Opts` is intended to be used as query parameters.
strcut tag `json` is not needed.

Reply:
ok